XESmartTarget.Core/Utils/XELFileCSVAdapter.cs

```diff
@@ -1,120 +1,124 @@
 ﻿using CsvHelper;
-using Microsoft.SqlServer.XEvent.Linq;
+using Microsoft.SqlServer.XEvent.XELite;
```

Comment:
I hope you tested this because it's a lot of code changes. Can we maybe have a look together?